src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (firstValue) => {
+    if (!Number.isInteger(firstValue)) {
```

Comment:
great move
but smth like `typeof firstValue === 'undefined'` will be more stable